modules/creating-the-vsphere-windows-vm-golden-image.adoc

```diff
@@ -19,7 +19,12 @@ You must use link:https://docs.microsoft.com/en-us/powershell/scripting/install/
 
 .Procedure
 
-. Create a new VM in the vSphere client using the Windows Server Semi-Annual Channel (SAC): Windows Server 20H2 ISO image that includes the link:https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351[Microsoft patch KB4565351]. This patch is required to set the VXLAN UDP port, which is required for clusters installed on vSphere. See the link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.hostclient.doc/GUID-77AB6625-F968-4983-A230-A020C0A70326.html[VMware documentation] for more information.
+. Create a new VM in the vSphere client using the Windows Server 2022 Long-Term Servicing Channel ISO image that includes the link:https://support.microsoft.com/en-us/topic/april-25-2022-kb5012637-os-build-20348-681-preview-2233d69c-d4a5-4be9-8c24-04a450861a8d[Microsoft patch KB5012637]. For example, you can use the following PowerShell command to download and install the patch:
++
+[source,posh]
+----
+PS C:\> Get-WindowsUpdate -Install -KBArticleId 'KB5012637'
```

Comment:
@saifshaikh48 I'm a little bit confused by this line and the block after line 74 (Install Microsoft container networking patch KB5012637)
So, here we are suggesting to install the KB patch via Get-WindowsUpdate, but later we are suggesting to install it again by downloading it directly? Or are these two different steps required to get the full patch installed?
I believe we did in our golden image what's indicated in line 26 and so far it has worked fine.

Reply:
Thanks for the clarification @saifshaikh48, I'll propose a modification to @mburke5678 then.

Reply:
@jfrancoa Thanks for catching this. It does seem redundant. Please let me know which of the two instances we should remove. Probably should remove the second instance, lines 74 to 90?

Reply:
@jfrancoa @saifshaikh48 Any thoughts on this ^^